src/storage/txn/commands/flush.rs

```diff
@@ -180,6 +188,16 @@ impl Flush {
                         assertion_failure = Some(e);
                     }
                 }
+                Err(crate::storage::mvcc::Error(
+                    box crate::storage::mvcc::ErrorInner::GenerationOutOfOrder(generation, lock),
+                )) => {
+                    info!(
+                        "generation in Flush is smaller than that in lock, ignore this mutation";
+                        "start_ts" => self.start_ts,
+                        "generation" => generation,
+                        "lock" => ?lock,
```

Comment:
Should the log be printted using red-act interfaces?

Reply:
Right. I've changed it from `LockInfo` to `Lock` and `Key`, which respects redact setting